coop_memberspace/security/ir.model.access.csv

```diff
@@ -11,8 +11,6 @@ access_shift_leave_type_memberspace,MemberSpace Access Shift Leave Type,coop_shi
 access_product_product_memberspace,MemberSpace Access Product Product,product.model_product_product,coop_memberspace.group_memberspace,1,0,0,0
 access_product_template_memberspace,MemberSpace Access Product Template,product.model_product_template,coop_memberspace.group_memberspace,1,0,0,0
 access_mail_template_memberspace,MemberSpace Access Mail Template,mail.model_mail_template,coop_memberspace.group_memberspace,1,0,0,0
-access_mail_attachment_custom_memberspace,MemberSpace Access Mail Attachment Custom,email_attachment_custom.model_mail_attachment_custom,coop_memberspace.group_memberspace,1,0,0,0
-access_memberspace_conversation,MemberSpace Access Memberspace Conversation,coop_memberspace.model_memberspace_conversation,coop_memberspace.group_memberspace,1,1,0,0
```

Comment:
`email_attachment_custom` has been replaced by https://github.com/druidoo/druidoo-addons/tree/12.0/mail_template_conditional_attachment

Functionally it's very similar.

We probably need to depend on this module, and write the security rules for the group_memberspace module